inspirehep/orcid/converter.py

```diff
@@ -276,9 +276,8 @@ def bibtex_citation(self):
                 )
             except Exception:
                 self._bibtex_citation = ""
-                logger.warning(
-                    "Bibtex citation serialization failed for"
-                    " recid={}".format(self.recid)
+                LOGGER.warning(
```

Comment:
`LOGGER.exception`? otherwise we'll never catch it

Reply:
:+1: